modules/exploits/windows/local/cve_2020_0787_bits_arbitrary_file_move.rb

```diff
@@ -0,0 +1,242 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core/post/common'
+require 'msf/core/post/windows/priv'
+require 'msf/core/post/windows/registry'
+require 'msf/core/exploit/exe'
+require 'msf/core/post/windows/filesystem'
+require 'msf/core/exploit/file_dropper'
+require 'msf/core/post/file'
```

Comment:
I don't think these should be necessary.

Reply:
Your right they are not. Removed